test/scripts/e2e_subs/rekey.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+date '+e2e_subs/rekey.sh start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+ACCOUNTB=$(${gcmd} account new|awk '{ print $6 }')
+
+algokey generate > ${TEMPDIR}/rekey
+mnemonic=$(grep 'Private key mnemonic:' < ${TEMPDIR}/rekey | sed 's/Private key mnemonic: //')
+ACCOUNTC=$(grep 'Public key:' < ${TEMPDIR}/rekey | sed 's/Public key: //')
```

Comment:
nit:
when using grep, you don't need to "<" into it. just pass the filename.
i.e.
```
grep 'Private key mnemonic:' ${TEMPDIR}/rekey
```